src/main/java/duke/datetime/DateTimeFormat.java

```diff
@@ -0,0 +1,7 @@
+package duke.datetime;
+
+public enum DateTimeFormat {
+    String,
```

Comment:
I like the use of an enum to keep track of valid & invalid formats. Much better than using integer status codes. Perhaps these should be in uppercase as it is an enum? I noticed a few other variables in a similar position.